compliance/aws/instances_without_fnm_agent/README.md

```diff
@@ -0,0 +1,92 @@
+The policy is a recommendation only policy, no action is taken during the Policy Escalation.
+
+### Functional Details
+The policy leverages the cloud API to get all current EC2 instances and the FlexNet Manager report (Custom view) API to get all AWS cloud instances with agent. It cross-checks the two lists to determine if any instances are running on the cloud that aren't known to FNM.  The policy matches the InstanceCloudID from FNMS and the instanceId from AWS. 
+
+### Prerequisites
+- FlexNet Manager
+- The following RightScale Credentials
+  - 'FNMS_API_Token'
+
+### Installation
+
+#### How to setup FlexNet Manager Custom View for this policy:
+
+1. Create a custom view in FlexNet manager that could look like this:
+![Alt text][FNMSReport]
+
+Click on Preview and filter.
+Select `Amazon Web Services` under `Inventory device` > `Hosted in`
+
+![Alt text][FilterFNMSReport]
+
+Once saved, note the report number in thr URL field :
+![Alt text][ReportNumber] you need it when activating the Policy for 'FNMS Report ID'.
+
+1. Setup the API Token in FNMS:
+    1. On the Account page - Select Create Account -> Service Account and fill in the form
+
+        ![Alt text][CreateServeceAccount]
+    1. IMPORTANT: When you hit save you will see a API Token.. This is the only time you will see it so you need to save it at this point
+    
+        ![Alt text][APIToken]
+    1. Add the new account to the Role ___Webservice___
+
+        ![Alt text][WebServiceRole]
+
+#### Cloud manager
+
+1. Create RightScale Credentials with values that match the FlexNet Manager API Token (Credential name: `FNMS_API_Token`) 
+
+### Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created
+- *Tags to ignore* - List of tags that will exclude EC2 instance from being evaluated by this policy. Multiple tags are evaluated as an 'OR' condition. Tag keys or key/value pairs can be listed. Example: 'test,env=dev'.
+- *FNMS Report ID* - FlexNet manager Custom View ID.
+
+### Required RightScale Roles
+
+- `policy_manager`
+- `credential_viewer`
+
+### AWS Required Permissions
+
+This policy requires permissions to describe EBS volumes.
+The Cloud Management Platform automatically creates two Credentials when connecting AWS to Cloud Management; AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. The IAM user credentials contained in those credentials will require the following permissions:
+
+```javascript
+ {
+    "Version": "2016-11-15",
+    "Statement":[{
+                  "Effect":"Allow",
+                  "Action":["ec2:DescribeInstances"],
+                  "Resource":"*"
+                }]
+ }
+```
+
+### Supported Clouds
+
+- AWS
+
+### Policy Actions
```

Comment:
Policy Action should be below Input Parameters

Reply:
Made required changes, Thanks